### DISCUSSION TITLE
Selanjutnya apa?

### DISCUSSION THREAD

**Discussion by billalxcode** [yandinovriandi/WMH#13] [Q&A]
Halo mas @yandinovriandi untuk pengembangan aplikasi ini, fitur selanjutnya apa yah?

**Comment by yandinovriandi** [comment]
crud di hotspot dulu bro spertinya, saya pending dulu , ngerjain ini dulu 

![image](https://github.com/yandinovriandi/WMH/assets/17920675/57c363eb-ab40-405f-84a1-bf6169bc13e4)
